src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const bestDiscount = 50;
+  const standardDiscount = 20;
+
+  if (days >= 7) {
+    return (dailyRate * days) - bestDiscount;
+  }
+
+  if (days >= 3) {
```

Comment:
Similarly, the number '3' is a literal and should be replaced with a constant that explains its meaning. For example, 'const SHORT_TERM_RENTAL = 3;'.